internal/app/apid/pkg/backend/apid.go

```diff
@@ -54,7 +54,7 @@ func (a *APID) String() string {
 // GetConnection returns a grpc connection to the backend.
 func (a *APID) GetConnection(ctx context.Context) (context.Context, *grpc.ClientConn, error) {
 	md, _ := metadata.FromIncomingContext(ctx)
-	md = metadata.Join(md, authz.RolesAsMetadata(authz.GetRoles(ctx)))
+	md = metadata.Join(md, authz.RolesAsMetadata(authz.GetRoles(ctx))) // TODO(rbac): duplicates?
```

Comment:
It seems that metadata is being duplicated there or in Local's method with the same name. It does not cause problems but means that my mental model of how everything works is wrong. I will fix it and add tests in the next PR.